tests/integration_tests/framework/docl.py

```diff
@@ -146,6 +149,11 @@ def upload_mock_license():
              '-c "{0}"'.format(INSERT_MOCK_LICENSE_QUERY)))
 
 
+def remove_license():
+    execute(('sudo -u postgres psql cloudify_db '
+             '-c "{0}"'.format(DELETE_CLOUDIFY_LICENSE_QUERY)))
```

Comment:
I think it is better to use 
`from integration_tests.framework import postgresql`
`postgresql.run_query("DELETE FROM licenses;")`